meta-resin-common/recipes-connectivity/networkmanager/files/NetworkManager.conf

```diff
@@ -0,0 +1,6 @@
+[main]
+dns=default
+plugins=keyfile
+
+[keyfile]
+unmanaged-devices=interface-name:docker0
```

Comment:
I think the main reason is because it is working when using NM without blacklist them. Why should they be blacklisted? What was the reason they got blacklisted in the first place when using connman?